enterprise_access/apps/subsidy_request/tasks.py

```diff
@@ -152,32 +152,33 @@ def send_learner_credit_bnr_admins_email_with_new_requests_task(
         policy_uuid, lc_request_config_uuid, enterprise_customer_uuid
 ):
     """
-    Task to send new learner credit request emails to admins.
+    Daily digest email to enterprise admins for Browse & Request learner credit requests.
 
-    This task can be manually triggered from browse_and_request when a new
-    LearnerCreditRequest is created. It will send the latest 10 requests in
-    REQUESTED state to enterprise admins.
+    Intended to be called once per day (via a scheduler). It aggregates all open learner credit
+    requests in REQUESTED state for the given policy and enterprise, and sends a summary email
+    containing up to the 10 most recent requests plus the total count of open requests.
 
     Args:
-        policy_uuid (str): subsidy access policy uuid identifier
-        lc_request_config_uuid (str): learner credit request config uuid identifier
-        enterprise_customer_uuid (str): enterprise customer uuid identifier
+        policy_uuid (str): Subsidy access policy UUID identifier.
+        lc_request_config_uuid (str): Learner credit request config UUID identifier.
+        enterprise_customer_uuid (str): Enterprise customer UUID identifier.
     Raises:
-        HTTPError if Braze client call fails with an HTTPError
+        HTTPError: If Braze client call fails with an HTTPError.
     """
-
     subsidy_model = apps.get_model('subsidy_request.LearnerCreditRequest')
     subsidy_requests = subsidy_model.objects.filter(
```

Comment:
[Reason for re-query] The task is re-queried at execution time. Between enqueue and run, requests can be approved/close,d or new ones arrive. If we “pass data” and skip the task’s query, we risk sending stale/empty or incorrect digests, and Celery retries would use outdated payloads.